src/components/homePage/homepage.css

```diff
@@ -0,0 +1,147 @@
+
+.context {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+  margin-top: 5%;
+  justify-content: center;
+  align-content: center;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(2, 0.4fr);
+  justify-content: center;
+  align-content: center;
+  width: 375px;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/79058364/217826455-b06fb502-f6f4-4064-8c34-ed92221dc9f1.png)
 

- [ ] According to the project requirements 🙃 Kindly As requested by the previous reviewer, make sure the hope page stick to the original design as much as possible 🎨 🖌️  ( You need to change your element Layout, background color, width, size ....)

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/217827407-e38e40c7-97be-441b-b147-0ba8a9191562.png)| ![Screenshot 2023-02-09 at 10 15 21 (2)](https://user-images.githubusercontent.com/64774936/217769278-651d6695-ed6d-4197-9584-53be3452a033.png)